Formula/u/uv.rb

```diff
@@ -0,0 +1,24 @@
+class Uv < Formula
+  desc "Extremely fast Python package installer and resolver, written in Rust"
+  homepage "https://github.com/astral-sh/uv"
```

Comment:
```suggestion
  homepage "https://astral.sh/"
```

Reply:
https://astral.sh actually does not feature the project at all (except for the aforementioned blog post). Wouldn't it be confusing to be directed to a site that doesn't mention the project?

Reply:
then should this get changed?

<img width="324" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/1580956/8074f4e8-4144-4c8a-a630-3bb137a33a48">

Reply:
Given that there is nothing on the homepage yet, I think it's fine to just use the repo for now.